tycho/conf/app-registry.yaml

```diff
@@ -52,6 +52,10 @@ repositories:
     # This is the Dockstore app prototype repository. GA4GH TRS in future?
     description: Main repository for Catalyst Apps and Tools
     url: https://github.com/helxplatform/app-support-prototype/raw/master/dockstore-yaml-proposals
+  argus_apps:
```

Comment:
@warrenstephens - There's actually an env variable [DOCKSTORE_APPS_BRANCH](https://github.com/helx-charts/appstore-chart/blob/94368ae2456efbeed68fc3f5fcbcf1758fcde5b3/values.yaml#L99), we can use to switch between the master and develop branches of app-support-prototype.

Reply:
@warrenstephens - There's an environment variable/chart value [DOCKSTORE_APPS_BRANCH](https://github.com/helx-charts/appstore-chart/blob/94368ae2456efbeed68fc3f5fcbcf1758fcde5b3/values.yaml#L99), we can use to switch between master and develop branches of app-support-prototype during the appstore deployment.

Reply:
Whether the value is "master" or "develop" the tycho code will cache the url response -- because it is using a CachedSession -- which causes subsequent changes in app-support-prototype to be ignored during development.

My "commit id" url approach makes tycho see all changes, and the DOCKSTORE_APPS_BRANCH logic could be removed as well.

Reply:
@muralikarthikk Actually, by using this "commit id" url approach then the CachedSession could be eliminated as well -- as all of the variables using http_session are singleton-style instantiations, yes?

Reply:
Or keep the CachedSession, as it may help if github is down???